internal/store/static.go

```diff
@@ -150,6 +155,28 @@ func (store *StaticStore) getPlugins(serverVersion string, includeEnterprisePlug
 			}
 		}
 
+		if platform != "" {
+			var bundle *model.PlatformBundleMetadata
+
+			// We don't want to edit the existing plugin's download_url. Only the response should be edited.
+			newRef := *storePlugin
```

Comment:
Could you elaborate a bit? What is the downside of editing the original variable `storePlugin`?

Reply:
I was mistakenly thinking this would be running ongoing with the plugin store in RAM, but it is running in a stateless lambda function, where any changes do not get committed back to the store. I will change it so it does not dereference.

Reply:
@hanzei After some testing, I've realized that I originally did this because the mutation persists in a local deployment. `plugins.json` must be kept in RAM between requests for the local deployment. I've changed it back to dereference.